packages/codegen-ui-react/lib/__tests__/__snapshots__/react-component-render-helper.test.ts.snap

```diff
@@ -254,8 +254,16 @@ exports[`react-component-render-helper buildConcatExpression should build concat
 
 exports[`react-component-render-helper buildContionalExpression operandType does not exist 1`] = `"(user?.age && user?.age > \\"18\\") ? \\"Vote\\" : \\"Sorry you cannot vote\\""`;
 
+exports[`react-component-render-helper buildContionalExpression operandType does not exist 2`] = `"(user?.age && user?.age > \\"true\\") ? \\"Vote\\" : \\"Sorry you cannot vote\\""`;
+
+exports[`react-component-render-helper buildContionalExpression operandType does not exist 3`] = `"(user?.age && user?.age > \\"dlo\\") ? \\"Vote\\" : \\"Sorry you cannot vote\\""`;
+
 exports[`react-component-render-helper buildContionalExpression operandType exists 1`] = `"(user?.age && user?.age > 18) ? \\"Vote\\" : \\"Sorry you cannot vote\\""`;
 
+exports[`react-component-render-helper buildContionalExpression operandType exists 2`] = `"(user?.age && user?.age > true) ? \\"Vote\\" : \\"Sorry you cannot vote\\""`;
```

Comment:
This isn't really the correct approach sematincally. Wouldn't `!!user?.age` make more sense?

Reply:
I didn't modify the part that creates this. I added to the test cases to hit all the previously missed branches. Also, can't it just be `user?.age > 18`?